### ISSUE TITLE
How can I converge three models of yaw, pitch, and roll to one model? 

### ISSUE THREAD

**Issue by gnnbest** [Cjiangbpcs/hopenet_mxJiang#3]
Hello, thanks for you share:
    I want to train the model by myself, can I use your code "train_gazenet_mxJiang.py" directly ? do I need to change some parameters?
   How can I converge three models of yaw, pitch, and roll to one model? can you upload your codes? thank you very much .

**Comment by Cjiangbpcs** [comment]
Yes, you can train the model using that .py file directly. The three models converging will not affect the accuracy etc. 

I am still in the process of cleaning up the scripts, will upload soon. But the existing code should work the same.
